.babelrc

```diff
@@ -10,6 +10,14 @@
     "transform-flow-strip-types"
   ],
   "env": {
+    "watch": {
+      "presets": [
+        ["es2015", {
```

Comment:
might be able to just do plugins commonjs here due to the inheriting of the top level presets/plugins but this works fine